app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from app.cafe import Cafe
+
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except (OutdatedVaccineError, NotVaccinatedError):
```

Comment:
OutdatedVaccineError and NotVaccinatedError are children of VaccineError, so you can simplify your code
```suggestion
        except VaccineError:
```